lib/sorcery/model.rb

```diff
@@ -140,6 +140,21 @@ def sorcery_config
         self.class.sorcery_config
       end
 
+      #  shows if user is logged in
+      def logged_in?
```

Comment:
This method can't be in core model, because all the fields that you're using here belong to `activity_logging` submodel. Therefore they need to be moved there